packages/svelte/src/internal/server/index.js

```diff
@@ -507,9 +507,12 @@ export { await_block as await };
 
 /** @param {any} array_like_or_iterator */
 export function ensure_array_like(array_like_or_iterator) {
-	return array_like_or_iterator?.length !== undefined
-		? array_like_or_iterator
-		: Array.from(array_like_or_iterator);
+	if (array_like_or_iterator) {
+		return array_like_or_iterator.length !== undefined
+			? array_like_or_iterator
+			: Array.from(array_like_or_iterator);
+	}
+	return [];
```

Comment:
Also it would be cool to add a test for this too 😉

Reply:
P.s. good catch

Reply:
The test I added will already fail for that in SSR mode.
Do we really need another test ?

Reply:
Oh i guess then it's fine.